app/models/concerns/csv_importable.rb

```diff
@@ -134,11 +134,30 @@ def process!
         end
 
       if Flipper.enabled?(:import_search_pds)
-        changesets.each do |patient_changeset|
-          PDSCascadingSearchJob.set(queue: :imports).perform_later(
-            patient_changeset,
-            queue: :imports
-          )
+        self.changesets.without_postcode.each do |patient_changeset|
```

Comment:
This it getting long, it's probably worth moving some of the work here into methods, maybe one to deal with changesets without postcodes and another to deal with those that have them.